config/redactor/Standard.json

```diff
@@ -1,5 +0,0 @@
-{
-  "buttons": ["html", "formatting", "bold", "italic", "unorderedlist", "orderedlist", "link", "image", "video", "file"],
-  "formatting": ["p", "blockquote", "h2", "h3", "h4", "h5"],
-  "plugins": ["fullscreen", "video", "table", "linkButton"]
-}
```

Comment:
This rename might break things in the CMS fields where things are tied to these filenames? eg:

![image](https://user-images.githubusercontent.com/394376/43143325-70489ec8-8f52-11e8-87cc-bd5b2bdfeb96.png)

Reply:
Tested this out and it defaults to…erm…the 'Default' profile so nothing _breaks_ but I will need to go through each redactor field and re-apply the configs.